tools/esp8266/User_Manual.md

```diff
@@ -18,11 +18,11 @@ The ahoy dtu will publish on the following topics
 |U_AC | 233.300|actual AC Voltage in Volt|
 |I_AC | 0.300 | actual AC Current in Ampere|
 |P_AC | 71.000| actual AC Power in Watt|
-|P_ACr | 21.200| actual AC reactive power in VAr|
-|Freq | 49.990|actual AC Frequency in 1/s|
-|Pct | 95.800|actual AC Power factor in %|
+|Q_AC | 21.200| actual AC reactive power in Var|
```

Comment:
ups, "var" ;-)

Reply:
fixed in [e57708b](https://github.com/lumapu/ahoy/pull/327/commits/e57708bd6e8dd9657253496380246cd5fb7db2e0)